build.gradle

```diff
@@ -135,7 +134,7 @@ subprojects {
                 }
             }
         }
-        if ((project.name.contains('samples') && !project.name.contains('kotlin')) || project.name.contains('benchmarks')) {
+        if ((project.name.contains('samples') && !project.name.contains('kotlin')) || project.name.contains('benchmarks') || project.name.contains('osgi-test')) {
```

Comment:
osgi test modules are not really java libraries anyway, and this avoids applying the JUnit dependency management we wold need to override in each.